### PR TITLE
For Ember Times No. 147: Add Writeup on Ember & Ember Inspector Release

### PR DESCRIPTION
**Pull request by jayjayjpg**

### CODE REVIEW

File: source/2020-05-08-the-ember-times-issue-147.md [hunk 1]
```diff
@@ -9,30 +9,31 @@ responsive: true
...
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Ember 3.18 is out 🎉](https://blog.emberjs.com/2020/05/05/ember-3-18-released.html)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+The latest version of **Ember.js**, **ember-data** and **ember-cli** is out [with its **3.18 release**](https://blog.emberjs.com/2020/05/05/ember-3-18-released.html)!
```
Comment (jaredgalanis) [inline source/2020-05-08-the-ember-times-issue-147.md]:
```suggestion
The latest versions of **Ember.js**, **ember-data** and **ember-cli** are out [with the **3.18 release**](https://blog.emberjs.com/2020/05/05/ember-3-18-released.html)!
```

File: source/2020-05-08-the-ember-times-issue-147.md [hunk 2]
```diff
@@ -9,30 +9,31 @@ responsive: true
...
+This new minor release version is full of bug fixes for the core libraries that power your Ember application, so be sure to upgrade soon.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+For more info on the current release schedule of Ember check out [our releases page](https://emberjs.com/releases/) and for infos on Ember v3.18, you can give the official release post [a read](https://blog.emberjs.com/2020/05/05/ember-3-18-released.html).
```
Comment (jaredgalanis) [inline source/2020-05-08-the-ember-times-issue-147.md]:
```suggestion
For more info on the current release schedule of Ember check out [our releases page](https://emberjs.com/releases/) and for information on Ember v3.18, you can give the official release post [a read](https://blog.emberjs.com/2020/05/05/ember-3-18-released.html).
```

File: source/2020-05-08-the-ember-times-issue-147.md [hunk 3]
```diff
@@ -9,30 +9,31 @@ responsive: true
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+Ember 3.18 is out 🎉,
+investigate arguments with Ember Inspector 🕵️‍♀️,
```
Comment (jaredgalanis) [inline source/2020-05-08-the-ember-times-issue-147.md]:
```suggestion
Investigate arguments with Ember Inspector 🕵️‍♀️,
```

**Review by jaredgalanis — APPROVED** [review]
Looks ✨✨✨! Just a few suggestions 😄

**Comment by jayjayjpg** [comment]
🙇‍♀️

**Pull request merged**